src/renderer/helpers/fp/eq.ts

```diff
@@ -0,0 +1,39 @@
+import * as RD from '@devexperts/remote-data-ts'
+import { Asset, BaseAmount } from '@thorchain/asgardex-util'
+import * as A from 'fp-ts/lib/Array'
+import * as Eq from 'fp-ts/lib/Eq'
+import * as O from 'fp-ts/lib/Option'
+
+import { AssetWithBalance, ApiError, AssetsWithBalance } from '../../services/wallet/types'
+
+export const eqOString = O.getEq(Eq.eqString)
+
+export const eqAsset: Eq.Eq<Asset> = {
+  equals: (x, y) => x.symbol === y.symbol
```

Comment:
im really not sure but here we compare only symbols. is there any chance that 2 different chains have the same symbol ?

Reply:
Good point, I will update it.